.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Your personal `gitignore` should not be included in a public PR.

Reply:
You can make a global gitignore file to ignore `.idea`, https://help.github.com/articles/ignoring-files/